tests/unit/synthDrivers/test_oneCore.py

```diff
@@ -1,34 +0,0 @@
-#tests/unit/synthDrivers/test_oneCore.py

```

Comment:
No; it was intentional. I got rid of the oneCore synth driver tests and instead added tests for localeNameToWindowsLCID , since the method I was testing in oneCore is really just a thin wrapper around the latter.